### PR TITLE
Paul/ui4 anims

### PR DESCRIPTION
**Pull request by paullinator**
### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [x] Yes
- [ ] No

### Dependencies

 none

### Description

https://github.com/EdgeApp/edge-login-ui-rn/assets/6079354/f132892c-ee9b-40f5-9536-31e34c6025a2

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206215218447921

### CODE REVIEW

File: src/components/common/EdgeAnim.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    if (type.endsWith('Left') || type.endsWith('Right')) {
+      builder = builder.withInitialValues({
+        transform: [{ translateX: translateDistance }]
+      })
+    } else {
+      builder = builder.withInitialValues({
+        transform: [{ translateY: translateDistance }]
+      })
+    }
+  }
```
Comment (samholmes) [inline src/components/common/EdgeAnim.tsx:97]: I don't quite understand this part. Why can't we have a fixed distance for all components? How are devs suppose to use this prop?

**Review by samholmes — CHANGES_REQUESTED** [review]
I just have a question perhaps to familiarize myself a bit with the new API. Clean PR tho!

File: src/components/common/EdgeAnim.tsx (hunk 1 above)
Comment (paullinator) [reply src/components/common/EdgeAnim.tsx:97]: If you look at the animations for the wallet list and tx list you’ll notice that rows near the top have a shorter animation distance than those at the bottom. This gives the effect of staggering the rows.

Comment (paullinator) [reply src/components/common/EdgeAnim.tsx:97]: As well, many animations have the effect of components coming together towards the center. This is achieved by giving components furthest from center having a larger animation distance.

**Review by samholmes — APPROVED** [review]

**Pull request merged**